Tasks/Ann_Valenda/task7/data.py

```diff
@@ -0,0 +1,112 @@
+import json
+
+file_name = 'data.json'
+with open(file_name, 'r') as f:
+    base = json.load(f)
```

Comment:
this works on module importing, which means that you read the file once and then process its content, would be better to access the data when needed